tests/debugger/do_eval_at.cmd

```diff
@@ -0,0 +1,16 @@
+eval_at 0
+eval_at 0 b
+n
+eval_at 0 b
+b do_eval_at.js:20
+n
+scopes
+eval_at 0 b
+eval_at 1 b
+eval_at 0 b=20
+eval_at 1 b=100
+n
+eval_at 0 a
+scopes
+eval_at 0 b
+c
```

Comment:
Please add negative test cases, like negative index, index over 65535, index over the max scope, etc.

Reply:
I've added some test cases.